src/plugins/data/public/ui/filter_bar/filter_options.tsx

```diff
@@ -161,19 +167,24 @@ class FilterOptionsUI extends Component<Props, State> {
         isOpen={this.state.isPopoverOpen}
         closePopover={this.closePopover}
         button={
-          <EuiButtonIcon
-            onClick={this.togglePopover}
-            iconType="filter"
-            aria-label={this.props.intl.formatMessage({
+          <EuiToolTip
```

Comment:
Do we know if EuiPopover will behave properly now that the button is wrapped in a tooltip?